test/functional/apps/visualize/group5/_tsvb_time_series.ts

```diff
@@ -225,14 +218,11 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
             const expectedFilterPills = ['0, win 7'];
             await visualBuilder.setMetricsGroupByTerms('bytes');
             await visChart.waitForVisualizationRenderingStabilized();
-            await header.waitUntilLoadingHasFinished();
```

Comment:
`waitForVisualizationRenderingStabilized` already has `waitUntilLoadingHasFinished` so I think it won't add stability.